jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/NavigationDrawersTests.java

```diff
@@ -82,37 +82,37 @@ public void imagesNavigationDrawerTest() {
         imagesNavigationDrawer.backgroundImage().has().sourcePath(backgroundImageLink);
     }
 
-    @Test
+    @Test(description = "Test checks that drawer appears as it's mini variant")
     public void miniVariantNavigationDrawerTest() {
         miniVariantNavigationDrawer.show();
         miniVariantNavigationDrawer.is().miniVariant();
     }
 
-    @Test
+    @Test(description = "Test checks that drawer is floating")
     public void floatingNavigationDrawerTest() {
         permanentAndFloatingNavigationDrawer.show();
         permanentAndFloatingNavigationDrawer.is().floating();
     }
 
-    @Test
+    @Test(description = "Test checks that drawer is permament")
     public void permanentNavigationDrawerTest() {
         permanentAndFloatingNavigationDrawer.show();
         permanentAndFloatingNavigationDrawer.is().permanent();
     }
 
-    @Test
+    @Test(description = "Test checks that drawer has expected height and width")
     public void heightNavigationDrawerTest() {
         imagesNavigationDrawer.show();
         imagesNavigationDrawer.has().height(300).and().width(300);
     }
 
-    @Test
+    @Test(description = "Test checks that drawer is at the right side")
     public void rightNavigationDrawerTest() {
         rightNavigationDrawer.show();
         rightNavigationDrawer.is().right();
     }
 
-    @Test
+    @Test(description = "Test checks overlay drawer has expected parameters")
     public void overlayTemporaryNavigationDrawerTest() {
```

Comment:
it's more about overlay feature, as it has color and opacity :)

Reply:
Yes, but this test is inside in Drawers test. I don't understand - this test is incorrect, or there should be other description about overlay(therefore, this test should be moved to OverlayTests, right?).

Reply:
thanks for clarification, everything is ok :)